_layouts/post-toc.html

```diff
@@ -13,7 +13,7 @@ <h3>Contents</h3>
         <section class="content">{{ content }}</section>
         <footer class="prev-next">{% include prev-next.html %}</footer>
               <hr>
-          <p><em>Have any feedback or questions? <a href="https://github.com/zapier/visual-builder/issues/new/choose" target="_blank">Let us know</a>.</em></p>
+		  <p><em>Need help? <a href="https://developer.zapier.com/contact" target="_blank">Tell us about your problem</a> and we’ll connect you with the right resource or contact support.</em></p>
```

Comment:
This looks fine HTML-wise, but the phrasing "Tell us about your problem and we'll connect you with the right resource or contact support" lands a bit odd with me grammatically. Technically, "we'll" is the subject, and "connect" and "contact" are the verbs associated with it, so it's essentially saying "**we'll** connect you with the right resource or **we'll** contact support." 

What if we simplified it?

> "Tell us about your problem and we'll connect you with the right resource."

I figure email support is one of our resources, so it seems reasonable to me.

Also, apologies for doing content review in a PR. :)

Reply:
@whymog, I just used the same phrasing that Megan provided from Figma for the Dev Platform as well. Do you want me to ask her about simplifying that phrase?

Reply:
Oh, no, it's cool. I don't want to second-guess our content design team. Thanks for clarifying!